### DISCUSSION TITLE
节点有什么问题，要告诉我哦

### DISCUSSION THREAD

**Discussion by free18** [free18/v2ray#2] [General]
有任何问题，可以留言告诉我，谢谢大家支持

**Comment by hong6666** [comment]
教程都是youtu.be的，看不了

**Reply by free18** [reply]
https://sscurl.com/#/register?code=xIFXozcD
看这里，注册账号进去，里面有各种系统教程

**Reply by aseefg** [reply]
clash配置失败

**Comment by Xjystudio** [comment]
readme第一个链接就404，请修复

**Reply by free18** [reply]
https://sscurl.com/#/register?code=xIFXozcD
看这里，注册账号进去，里面有各种系统教程

**Comment by happysugar02** [comment]
用不了啊

**Reply by fkunasa** [reply]
能用节点变少了

**Comment by rookievips** [comment]
没一个能用的
